README.md

```diff
@@ -57,10 +57,10 @@ Today, we'll introduce a third: _rebasing_.
 
 ### Git Rebase, in Pictures
 
-Suppose that you have two branches in your project, `master` and `feature`,
+Suppose that (in addition to master) you have two branches in your project, `dev` and `feature`,
```

Comment:
I agree, I only went with dev because development obscured some of the diagram with its length. I'll open an issue for it now.